libnopegl/src/block.c

```diff
@@ -152,6 +160,14 @@ int ngli_block_add_field(struct block *s, const char *name, int type, int count)
 {
     ngli_assert(s->layout != NGLI_BLOCK_LAYOUT_UNKNOWN);
 
+    if (count == NGLI_BLOCK_VARIADIC_COUNT)
+        ngli_assert(s->layout == NGLI_BLOCK_LAYOUT_STD430);
```

Comment:
You can have variadic declarations with the STD140 layout.

Reply:
Removed, although I have reservations because of the previously mentioned issue.